Python/pylifecycle.c

```diff
@@ -83,6 +83,8 @@ int _Py_UnhandledKeyboardInterrupt = 0;
  * found without any external information. This is meant to ease access to the
  * interpreter state for various runtime debugging tools, but is *not* an
  * officially supported feature */
+_Py_COMP_DIAG_PUSH
```

Comment:
You should move and update ``/* Suppress deprecation warning for PyBytesObject.ob_shash */`` comment, no?

Reply:
Agreed.